plugin/storage/es/spanstore/schema.go

```diff
@@ -0,0 +1,183 @@
+               "type":"long"
+            },
+            "flags":{
+               "type":"integer"
+            },
+            "logs":{
+               "properties":{
+                  "timestamp":{
+                     "type":"long"
+                  },
+                  "tags":{
+                     "type":"nested",
+                     "dynamic":false,
+                     "properties":{
+                        "key":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        },
+                        "value":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        },
+                        "tagType":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        }
+                     }
+                  }
+               }
+            },
+            "process":{
+               "properties":{
+                  "serviceName":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "tags":{
+                     "type":"nested",
+                     "dynamic":false,
+                     "properties":{
+                        "key":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        },
+                        "value":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        },
+                        "tagType":{
+                           "type":"keyword",
+                           "ignore_above":256
+                        }
+                     }
+                  }
+               }
+            },
+            "references":{
+               "type":"nested",
+               "dynamic":false,
+               "properties":{
+                  "refType":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "traceID":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "spanID":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  }
+               }
+            },
+            "tags":{
+               "type":"nested",
+               "dynamic":false,
+               "properties":{
+                  "key":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "value":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  },
+                  "tagType":{
+                     "type":"keyword",
+                     "ignore_above":256
+                  }
+               }
+            }
+         }
+      },
+      "service":{
```

Comment:
It's not advised to use multiple "types" in a single index. Actually if I remember correctly it is going to be deprecated or removed in ES 6.x. The reason is that those types are "merged" and the indexed documents have fields from both types, but fields from the other type are empty.